ai-platform-unified/notebooks/unofficial/sdk/metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Please inject REGION instead of hardcording to us-central. See testing doc for "substitutions".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Updated this per https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb</p>